CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -4,7 +4,7 @@ Changelog for the ZDM Proxy, new PRs should update the `unreleased` section.
 
 When cutting a new release, update the `unreleased` heading to the tag being generated and date, like `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unreleased` entries.
 
-## Unreleased
+## Released
```

Comment:
This was correct. We use the `Unreleased` section when we have things merged in `main` that have not been released yet.

Reply:
now since it was released, https://github.com/datastax/zdm-proxy/releases/tag/v2.1.0 shouldn't we move it to a released section?

Reply:
there is no "Released" section. There is one section per release and one section for everything Unreleased (no version exists)